common/scala/src/main/scala/whisk/core/entity/Parameter.scala

```diff
@@ -53,26 +53,30 @@ protected[core] class Parameters protected[entity] (
             name.size + value.size
     }.foldLeft(0 B)(_ + _)
 
-    protected[entity] def ++(p: (ParameterName, ParameterValue)) = {
+    protected[entity] def +(p: (ParameterName, ParameterValue)) = {
```

Comment:
FWIW regarding scala patterns: The `apply` method for `Parameters` returns an `Option` vs. with `Map` `apply` is the dangerous version which will throw an exception. Having a `get` method instead could be desirable? (I stumbled over this at least once)